src/components/misc/named.ts

```diff
@@ -0,0 +1,7 @@
+import type { Comp, NamedComp } from "@/types";
```

Comment:
Also if you want you can put the types here, then do 

```js
// types.ts
import type { NamedComp } from "@/components"
export { NamedComp }
```

Reply:
I actually wanted name instead of named. But it wouldn't let me. I did the NamedComp originally inside the component file, however I wanted to see an example of the new convention first, so it would be the same.